cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -815,9 +815,9 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
         classes.add(inputFormatClass);
 
         // If it is StreamInputFormat, also add the StreamEventCodec class as well.
-        if (AbstractStreamInputFormat.class.isAssignableFrom(inputFormatClass)) {
+        if (MapReduceStreamInputFormat.class.isAssignableFrom(inputFormatClass)) {
```

Comment:
Hmm, no not required. Actually, I could have just left it as AbstractStreamInputFormat (which was actually added in this PR) but since it is used in MapReduceRuntimeService, I thought we could just safely use this class.